src/WalletProviderTester.ts

```diff
@@ -1,42 +1,76 @@
 import IWalletProvider from "bch-wallet-bridge-provider-interface/lib/IWalletProvider"
+import Utxo from "bch-wallet-bridge-provider-interface/lib/entities/Utxo"
 
 class WalletProviderTester {
     constructor(public walletProvider: IWalletProvider) {}
 
     //
     // getVersion
     //
+    public async testGetVersion(): Promise<boolean> {
+        const version = await this.walletProvider.getVersion()
+        if (typeof version !== "string") {
+            throw new Error("The return value is invalid.")
+        }
+        return true
+    }
 
     //
     // getAddresses
     //
-    public testGetAddresses(): Promise<boolean> {
-        return this.walletProvider
-        .getAddresses(0, 10, 0, undefined)
-        .then((addresses) => {
-            const address = addresses[0]
-            if (typeof address !== "string") {
-              throw new Error("The return value is invalid.")
-            }
-            return addresses.length === 10
-          })
+    public async testGetAddresses(): Promise<boolean> {
+        const addresses = await this.walletProvider.getAddresses(0, 10, 0, undefined)
+        const address = addresses[0]
+        if (typeof address !== "string") {
```

Comment:
You can write as one liner.
`addresses.length !== 10 || (typeof addresses[0] !== "string")`

Reply:
I know but error message would better be different.

Reply:
Okay, great